index.html

```diff
@@ -0,0 +1,136 @@
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.typekit.net/uzf0rar.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <title>Reneweable Energy Conference</title>
+</head>
+<body>
+    <section class="background">
+        <nav class ="desktopnav">
+            <div class="space"></div>
+            <ul class="socialmedia">
+                <li><a href="#"><i class=" fa fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class=" fa fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class ="desktop2nav">
+            <div class="space2"><img src="image/logo-EHS.png" class="desktoplogo"></div>
+            <ul class="socialmedia2">
+                <li><a href="about.html" id="about">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" id="Campaign">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="meuncomponents">
+            <button class="menubtn" type="submit"><img src="image/hamburger.JPG" id ="menu"></button>
+            <div class="menuspace"></div>
+        </div>
+        <div class="popup">
+            <a href=""> About </a>
+        </div>
+        <div class="header-text">
+            <h2>"Hello! Sharing World"</h2>
+                <h1>National Reneweable Energy Submit 2023</h1>
+           </div>
+           <div class="header-paragraph">
+            <p>A global submit where experts in the field of Reneweable Energy Technologies from all over the world would converge to discuss the latest developments in Lagos State Nigeria, March 2023</p>
+           </div>
+
+           <div class="header-footer">
+            <h2>2023.03.01(WED)~02(THU)</h2>
+            <p> @National art theater iganmu lagos</p>
+           </div>
+        </section>
+    <section class="main-program">
+            <div class="main-heading">
+                <h2>Main Program</h2>
+                <hr>
+            </div>
+            <div class= "main-itemcontainer">
+            <div class="main-item">
+                <div class="mainIMGH">
+                    <img class="main-image" src="image/speaker.png">
+                    <h2>Lecture</h2>
+                </div>
+                
+                <p>Listen to various experts in Reneweable Energy share that ideas on the future of Reneweable</p>
+            </div>
+            <div class="main-item">
+                <div class="mainIMGH">
+                <img class="main-image" src="image/Projector.png">
+                <h2>Exhibition</h2>
```

Comment:
- [ ] You have done incredibly well by using img tag. However, you should ensure that the alt has a text. This is because alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see. Kindly ensure to make your page more accessible by implementing the alt text 👍 .